### ISSUE TITLE
`python3 run.py -c 10` raises `TypeError`

### ISSUE THREAD

**Issue by mrtpk** [Belval/TextRecognitionDataGenerator#114]
After cloning, ran `python3 run.py -c 10` produced `TypeError`. Below is the error stack. Please help.

multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "anaconda3/envs/dev/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "anaconda3/envs/dev/lib/python3.5/site-packages/trdg/data_generator.py", line 21, in generate_from_tuple
    cls.generate(*t)
TypeError: generate() takes 24 positional arguments but 25 were given
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "run.py", line 407, in <module>
    main()
  File "run.py", line 391, in main
    total=args.count,
  File "anaconda3/envs/dev/lib/python3.5/site-packages/tqdm/std.py", line 1104, in __iter__
    for obj in iterable:
  File "anaconda3/envs/dev/lib/python3.5/multiprocessing/pool.py", line 731, in next
    raise value
TypeError: generate() takes 24 positional arguments but 25 were given
  0%|                                                                                                                    | 0/1 [00:00<?, ?it/s]

**Comment by Belval** [comment]
Here is what I did:

1. `git clone git@github.com:Belval/TextRecognitionDataGenerator.git`
2. `virutalenv --python python3 env/`
3. `source env/bin/activate`
4. `pip3 install -r requirements.txt`
5. `cd trdg/`
6. `python3 run.py -c 10`

And it worked.

But from the error, it seems like you might have an older version that is missing a parameter (and somehow only updated partially). If you do a new clean clone, does it work with the steps I listed above?
